scripts/integration/kafka/compose.yaml

```diff
@@ -2,34 +2,33 @@ version: '3'
 
 services:
   zookeeper:
-    image: docker.io/wurstmeister/zookeeper:${CONFIG_VERSION}
+    image: docker.io/confluentinc/cp-zookeeper:${CONFIG_VERSION}
     ports:
     - 2181:2181
+    environment:
+    - ZOOKEEPER_CLIENT_PORT=2181
   kafka:
-    image: docker.io/wurstmeister/kafka:2.13-2.6.0
+    image: docker.io/confluentinc/cp-kafka:7.6.1
     depends_on:
     - zookeeper
     environment:
     - KAFKA_BROKER_ID=1
     - KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181
+    - ZOOKEEPER_SASL_ENABLED=false
     - KAFKA_LISTENERS=PLAINTEXT://:9091,SSL://:9092,SASL_PLAINTEXT://:9093
     - KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://kafka:9091,SSL://kafka:9092,SASL_PLAINTEXT://kafka:9093
     - KAFKA_SSL_KEYSTORE_TYPE=PKCS12
-    - KAFKA_SSL_KEYSTORE_LOCATION=/certs/kafka.p12
-    - KAFKA_SSL_KEYSTORE_PASSWORD=NOPASS
-    - KAFKA_SSL_TRUSTSTORE_TYPE=PKCS12
```

Comment:
Tests aren't actually providing a valid client certificate so I removed the truststore bits and set `KAFKA_SSL_CLIENT_AUTH=none`.